index.js

```diff
@@ -400,6 +400,12 @@ function fastifyMultipart (fastify, options, done) {
         return
       }
 
+      if (opts.throwFileSizeLimit || options.throwFileSizeLimit) {
```

Comment:
I would consider those default to `true`. This seems a bug in the original implementation.

Reply:
We can release a `semver-minor` with false, then go with a `semver-major` with true in another PR.
So, it would not break someone code and allow them to use this feature.

Reply:
agree with @climba03003 , this bug was there for a certain amount of time so maybe people would break their code if the upgrade is in `semver-minor`

Reply:
side node `opts` and `options` are kind of confusing. Might be worth to rename those properly.

Reply:
I agree that `opts` and `options` are too confusing when I scanning the code. I will take a time to improve this same as what I have done for fastify-swagger. I will mark it as `TODO`. 

The big `refactor` will comes with `semver-major` and the default behavior change.

Reply:
I'm happy to bump the major and have a good default.